expo/gbe/cms_app.py

```diff
@@ -2,9 +2,10 @@
 from cms.apphook_pool import apphook_pool
 from django.utils.translation import ugettext_lazy as _
 
-#this is largely a proof of concept right now, I haven't found this this was useful.
+
+# this is largely a proof of concept right now,
```

Comment:
Is this serving any purpose? The comment indicates that it isn't, in which case it should be deleted.